src/surrogates/inc/SurrogateBuilderBase.h

```diff
@@ -41,7 +44,7 @@ namespace QUESO
 
     virtual ~SurrogateBuilderBase(){};
 
-    virtual double evaluate_model( const V & domainVector ) const =0;
+    virtual void evaluate_model( const V & domainVector, std::vector<double>& values ) =0;
```

Comment:
As I mentioned before, if I only have one data set I'd have to populate a vector of length 1, but maybe this isn't a big deal.